src/ilasm/CMakeLists.txt

```diff
@@ -25,6 +25,21 @@ set(ILASM_SOURCES
   prebuilt/asmparse.cpp
 )
 if(WIN32)
+  set(ILASM_HEADERS
```

Comment:
A nit - to make it consistent with other similar PRs of yours, can you please set the ILASM_HEADERS and ILDASM_HEADERS unconditionally and only make adding them to the ILASM_SOURCES conditional?